index.html

```diff
@@ -249,6 +252,87 @@ <h2>Introduction</h2>
         </li>
       </ul>
 
+      <section class="informative">
+        <h3>Intended Audience</h3>
+
+        <p>
+The intended audience for the Verifiable Credentials Data Model 1.0 W3C Recommendation (Recommendation) 
+is a broad range of professionals interested in furthering
+the application of verifiable credentials technology for use in software apps, agents, and services.
+        </p>
+        <p>
+The primary audience includes software architects, application developers, and user experience (UX)
+specialists;
+as well as other people involved in a broad range of standards efforts related to decentralized identity,
+verifiable credentials, and secure storage.
+        </p>
+      </section>
+
+      <section class="informative">
+        <h3>Purpose</h3>
+
+        <p>
+The purpose of this Recommendation is to assist the intended audience by providing a deeper understanding
+of what a verifiable credential is as well as how to use verifiable credentials technology
+when designing and creating new software apps, agents, and services. 
```

Comment:
I think the purpose of the spec is to tell readers how to express digital credentials in a standard format (and things along those lines), not tell people how to use VC technology. I'm also not sure that saying that the spec should give people a "deeper understanding of what a verifiable credential is" makes sense -- the spec defines what a VC is. We could say that the purpose of the VC spec is to define what a VC is ... but that's also perhaps unnecessary or self-explanatory.

Reply:
> We could say that the purpose of the VC spec is to define what a VC is

Good feedback. The PR was intended to be a first draft - to be improved along these lines.

Aside: My original wording assumed the reader would be arriving to read the Documentation with some level of awareness of what a VC is.  I have no problem simplifying this sentence.  I'll wait and try to incorporate all of the changes at the same time.